pom.xml

```diff
@@ -7,6 +7,18 @@
     <groupId>group</groupId>
     <artifactId>1.0</artifactId>
     <version>1.0-SNAPSHOT</version>
```

Comment:
You have used MVC pattern to organize your code. Do you think it suites there? Controller shouldn't hold logic, controllers parse user input and redirect manipulation with objects/logic to model layer.

Reply:
done - removed controller package at all 
All methods are incuded to the related classes and located in model package
view is only for menu displaying